### PR TITLE
docs: Update the roadmap for 1.10

### PR DESCRIPTION
**Pull request by travisn**
**Description of your changes:**
Add planned features to the list for 1.10 planning.

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/development-flow.html#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -8,21 +8,25 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.9
+## Rook Ceph 1.10
 
-The following high level features are targeted for Rook v1.9 (early April 2022). For more detailed project tracking see the [v1.9 board](https://github.com/rook/rook/projects/24).
+The following high level features are targeted for Rook v1.10 (early August 2022). For more detailed project tracking see the [v1.10 board](https://github.com/rook/rook/projects/26).
+
+* Remove support for Ceph Octopus (support remains for Pacific and Quincy) [#10338](https://github.com/rook/rook/issues/10338)
```
Comment (galexrt) [inline ROADMAP.md:15]: Even with the documentation restructure/ rework being partially done today, should we add it as we are still working on further rewrites of the documentation?

Comment (travisn) [reply ROADMAP.md:15]: The roadmap is really for capturing changes in future releases. Anything we are going to backport to the current release I don't think needs to be captured here. Anytime we update the docs, we're likely going to backport it though unless it's a major restructure. But maybe I'm not aware of all the plans you have for the docs, so we should chat more. :)

File: ROADMAP.md [hunk 2]
```diff
@@ -8,21 +8,25 @@ We hope that the items listed below will inspire further engagement from the com
...
+* Create tool to analyze cluster health and advise on resolving common health issues [Krew #32](https://github.com/rook/kubectl-rook-ceph/issues/32)
```
Comment (subhamkrai) [inline ROADMAP.md:16]:
```suggestion
* Add support in kubectl-rook-ceph krew plugin to analyze cluster health and advise on resolving common health issues [Krew #32](https://github.com/rook/kubectl-rook-ceph/issues/32)
```
`create tool` to be sounds like we need to create a new tool

File: ROADMAP.md [hunk 3]
```diff
@@ -8,21 +8,25 @@ We hope that the items listed below will inspire further engagement from the com
...
+* Check for existing subvolumes before allowing a filesystem to be uninstalled [#9915](https://github.com/rook/rook/pull/9915)
+* Use more specific cephx accounts to better differentiate the source of configuration changes [#10169](https://github.com/rook/rook/issues/10169)
```
Comment (subhamkrai) [inline ROADMAP.md:18]:
```suggestion
* Use more specific cephx accounts to better differentiate the source of ceph configuration changes [#10169](https://github.com/rook/rook/issues/10169)
```
suggestio: how to about adding `ceph configuration changes`? or feel to skip this change

File: ROADMAP.md [hunk 4]
```diff
@@ -8,21 +8,25 @@ We hope that the items listed below will inspire further engagement from the com
...
 
 ## Themes
 
 The general areas for improvements include the following, though may not be committed to a release.
 
 * Add alpha support for COSI (Container object storage interface) with K8s 1.24 [#7843](https://github.com/rook/rook/issues/7843)
-* iSCSI gateway deployment [#4334](https://github.com/rook/rook/issues/4334)
-* Enable the admission controller by default [#6242](https://github.com/rook/rook/issues/6242)
 * OSD encryption key rotation [#7925](https://github.com/rook/rook/issues/7925)
 * Simplify metadata backup and disaster recovery
```
Comment (subhamkrai) [inline ROADMAP.md:31]: do we have any issues tracking this or is it just common in every release?

File: ROADMAP.md [hunk 5]
```diff
@@ -8,21 +8,25 @@ We hope that the items listed below will inspire further engagement from the com
...
+The following high level features are targeted for Rook v1.10 (early August 2022). For more detailed project tracking see the [v1.10 board](https://github.com/rook/rook/projects/26).
+
+* Remove support for Ceph Octopus (support remains for Pacific and Quincy) [#10338](https://github.com/rook/rook/issues/10338)
+* Create tool to analyze cluster health and advise on resolving common health issues [Krew #32](https://github.com/rook/kubectl-rook-ceph/issues/32)
+* Check for existing subvolumes before allowing a filesystem to be uninstalled [#9915](https://github.com/rook/rook/pull/9915)
+* Use more specific cephx accounts to better differentiate the source of configuration changes [#10169](https://github.com/rook/rook/issues/10169)
+* Automate node fencing for application failover in some scenarios [#1507](https://github.com/rook/rook/issues/1507)
+* Object Store
+  * Service account authentication with the Vault agent [#9872](https://github.com/rook/rook/pull/9872)
+  * Support for AWS Server Side Encryption (SSE) [#10318](https://github.com/rook/rook/pull/10318)
```
Comment (thotz) [inline ROADMAP.md:22]: I am planning to include some improvements for rgw multisite in 1.10 like https://github.com/rook/rook/pull/10323

File: ROADMAP.md (hunk 4 above)
Comment (travisn) [reply ROADMAP.md:31]: There should be one, I need to link it...

**Review by thotz — APPROVED** [review]

**Review by subhamkrai — APPROVED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Review by galexrt — APPROVED** [review]

File: ROADMAP.md [hunk 6]
```diff
@@ -8,23 +8,28 @@ We hope that the items listed below will inspire further engagement from the com
...
+
+* Remove support for Ceph Octopus (support remains for Pacific and Quincy) [#10338](https://github.com/rook/rook/issues/10338)
+* Add command to the [krew plugin](https://github.com/rook/kubectl-rook-ceph) to analyze cluster health and advise on resolving common health issues [Krew #32](https://github.com/rook/kubectl-rook-ceph/issues/32)
+* Check for existing subvolumes before allowing a filesystem to be uninstalled [#9915](https://github.com/rook/rook/pull/9915)
+* Use more specific cephx accounts to better differentiate the source of Ceph configuration changes [#10169](https://github.com/rook/rook/issues/10169)
+* Automate node fencing for application failover in some scenarios [#1507](https://github.com/rook/rook/issues/1507)
+* Object Store
+  * Service account authentication with the Vault agent [#9872](https://github.com/rook/rook/pull/9872)
+  * Support for AWS Server Side Encryption (SSE) [#10318](https://github.com/rook/rook/pull/10318)
+  * Improvements to RGW Multisite configuration [#10323](https://github.com/rook/rook/pull/10323)
```
Comment (parth-gr) [inline ROADMAP.md:24]: Can we add 
`Automate the process of StorageClass creation and add multitenancy support for external Cluster` Reference: https://github.com/rook/rook/pull/10278

**Review by parth-gr — APPROVED** [review]

**Pull request merged**